lib/app_component.dart

```diff
@@ -39,6 +39,6 @@ import 'package:client/admin/admin.dart';
       path: '/tutorial/:id',
       name: 'Tutorial',
       component: TutorialPageComponent),
-  const Route(path: '/admin', name: 'Admin', component: AdminPageComponent),
+  const Route(path: '/admin/...', name: 'Admin', component: AdminPageComponent),
```

Comment:
What's this for?

Reply:
It indicated a non-terminal route and it's required for AdminPage to have child routes (/admin/add-tutorial and /modify-tutorial/:id)